philly/setup.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
For the stuff about Philly, can we either leave it out of the repository or make it more generic if possible (e.g. scripts for running on a Docker container)?